code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/builders/TimerNotificationBuilder.kt

```diff
@@ -165,12 +165,15 @@ internal object TimerNotificationBuilder {
      */
     private fun createIntent(template: TimerPushTemplate): Intent {
         val intent = AEPPushNotificationBuilder.createIntent(PushTemplateConstants.IntentActions.TIMER_EXPIRED, template)
-        intent.putExtra(PushTemplateConstants.PushPayloadKeys.TEMPLATE_TYPE, template.templateType?.value)
         intent.putExtra(TimerKeys.ALTERNATE_TITLE, template.alternateTitle)
         intent.putExtra(TimerKeys.ALTERNATE_BODY, template.alternateBody)
         intent.putExtra(TimerKeys.ALTERNATE_EXPANDED_BODY, template.alternateExpandedBody)
         intent.putExtra(TimerKeys.ALTERNATE_IMAGE, template.alternateImage)
         intent.putExtra(TimerKeys.TIMER_COLOR, template.timerColor)
```

Comment:
We can remove all the Above lines that adds keys to the Intent.. They are repetitive 
```
 intent.putExtra(TimerKeys.ALTERNATE_TITLE, template.alternateTitle)
        intent.putExtra(TimerKeys.ALTERNATE_BODY, template.alternateBody)
        intent.putExtra(TimerKeys.ALTERNATE_EXPANDED_BODY, template.alternateExpandedBody)
        intent.putExtra(TimerKeys.ALTERNATE_IMAGE, template.alternateImage)
        intent.putExtra(TimerKeys.TIMER_COLOR, template.timerColor)
 ```

Reply:
We will need to do this for all builders. I've added todos, will get to it post snapshot release clean up